src/wasm.h

```diff
@@ -90,6 +90,9 @@ struct Name : public cashew::IString {
 
 // An index in a wasm module
 typedef uint32_t Index;
+inline Index toIndex(size_t index) {
```

Comment:
Do you mean add an assert() within toIndex()? Yes, if max (size) is passed as an additional parameter, an assert could be added. Should I do that?